### PR TITLE
Changelogs for Digital Twin Manager 1020.1.11

### PR DESCRIPTION
**Pull request by lra-sag**
This PR contains the changelogs for Digital Twin Manager 1020.1.11

### CODE REVIEW

File: content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+    - value: "component-Tl88RYb4A"
+      label: "Digital Twin Manager"
+build_artifact:
+    - value: "tc-wYIY0MBDO"
+      label: "dtm"
+ticket: "CTM-1407"
+version: "1020.1.11"
+---
+Added Wind Turbine Model and Sample Power Plant Model as a default sample asset model in Asset model samples screen.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md:17]:
```suggestion
In the Asset model samples screen, the "Wind Turbine Model" and the "Sample Power Plant Model" have been added as default sample asset models.
```
What is the exact spelling of the models in the UI? Probably it should be "Wind turbine model" and "Sample power plant model" as we generally use sentence capitalization? 

Isn´t the Asset model samples screen a page? Then it must be **Asset model samples** page.

File: content/change-logs/application-enablement/dtm-1020.1.11-update-label-capitalization.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "Update label capitalization"
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/dtm-1020.1.11-update-label-capitalization.md:3]:
```suggestion
title: "Updated label capitalization"
```

File: content/change-logs/application-enablement/dtm-1020.1.11-update-label-capitalization.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+    - value: "component-Tl88RYb4A"
+      label: "Digital Twin Manager"
+build_artifact:
+    - value: "tc-wYIY0MBDO"
+      label: "dtm"
+ticket: "CTM-1397"
+version: "1020.1.11"
+---
+Fixed inconsistent capitalization of labels, error messages and info messages to a uniform style throughout DTM.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/dtm-1020.1.11-update-label-capitalization.md:17]:
```suggestion
Inconsistent capitalization of labels, error messages and info messages has been unified throughout the Digital Twin Manager application.
```

File: content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "Add default sample asset models"
+product_area: "Application enablement & solutions"
+change_type:
+    - value: "change-VSkj2iV9m"
+      label: "Fix"
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md:7]: Is this really a fix? Sounds more like an improvement.

**Review by pavanshivajirao — APPROVED** [review]

File: content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+    - value: "component-Tl88RYb4A"
+      label: "Digital Twin Manager"
+build_artifact:
+    - value: "tc-wYIY0MBDO"
+      label: "dtm"
+ticket: "CTM-1407"
+version: "1020.1.11"
+---
+In the **Asset model samples** screen, the "Wind Turbine Model" and the "Sample Power Plant Model" have been added as default sample asset models.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md:17]:
```suggestion
In the **Asset model samples** page, the "Wind Turbine" model and the "Sample Power Plant" model have been added as default sample asset models.
```

File: content/change-logs/application-enablement/dtm-1020.1.11-update-label-capitalization.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+component:
+    - value: "component-Tl88RYb4A"
+      label: "Digital Twin Manager"
+build_artifact:
+    - value: "tc-wYIY0MBDO"
+      label: "dtm"
+ticket: "CTM-1397"
+version: "1020.1.11"
+---
+Inconsistent capitalization of labels, error messages and info messages has been unified throughout the Digital Twin Manager application.
```
Comment (MWindel) [inline content/change-logs/application-enablement/dtm-1020.1.11-update-label-capitalization.md:17]:
```suggestion
Inconsistent capitalization of labels, error messages and info messages has been unified throughout the Digital twin manager application.
```

File: content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "Added default sample asset models"
+product_area: "Application enablement & solutions"
+change_type:
+    - value: "change-VSkj2iV9m"
```
Comment (lra-sag) [inline content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md:6]:
```suggestion
    - value: "change-2c7RdTdXo4"
```

File: content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md (hunk 1 above)
Comment (rahultsag) [reply content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md]: i will suggest this description here: In the Asset Model Samples page, we've added two new sample models for a Wind Turbine and a Solar Power Plant. These samples serve as  an example and help you quickly get started with your own asset modeling.

**Review by rahultsag — CHANGES_REQUESTED** [review]
i will suggest this description here: In the Asset Model Samples page, we've added two new sample models for a Wind Turbine and a Solar Power Plant. These samples serve as  an example and help you quickly get started with your own asset modeling.

File: content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md (hunk 1 above)
Comment (Vagdevi1998) [reply content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md]: Modified the description as below: 
In the **Asset model samples** page, we've added two new sample models for "Wind Turbine" and "Solar Power Plant". These samples serve as an example and help you quickly get started with your own asset modeling.

File: content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md (hunk 4 above)
Comment (Vagdevi1998) [reply content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md]: It is an improvement

**Review by rahultsag — APPROVED** [review]

File: content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+---
+date: "2024-07-04"
+title: "Added default sample asset models"
```
Comment (MWindel) [inline content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md:3]:
```suggestion
title: "Added default sample asset models to facilitate asset modeling"
```

File: content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
+---
+date: "2024-07-04"
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md:2]: The date field should be left empty initially and filled in later. Or has DTM 1020.1.11 been deployed to Zone 1 on July 4th?

Comment (Vagdevi1998) [reply content/change-logs/application-enablement/dtm-1020.1.11-add-default-sample-asset-models.md:2]: DTM 1020.1.11 has been deployed to Zone 1 on July 4th.

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**